app/src/screens/Dialer/ToggleClientInputButton.tsx

```diff
@@ -17,7 +17,7 @@ const ToggleClientInputButton: React.FC<Props> = ({
   <Button disabled={disabled} size={96} onPress={onPress}>
     <Image source={TypeSource} resizeMode="contain" style={styles.image} />
     <Text style={styles.text}>
-      {recipientType === 'number' ? 'Number' : 'Client'}
+      {recipientType === 'number' ? 'Client' : 'Number'}
```

Comment:
I don't get this change. Why switch the two?

Reply:
Currently on main we have this: 
![dialer](https://user-images.githubusercontent.com/22135968/223575482-53f733b4-c828-4dd1-b100-7cc945a52c9f.jpg)

But I believe this is incorrect, since if you are using number dialpad and click the "number" toggle the "client" toggle will be triggered.

This is the figma:
<img width="229" alt="Screen Shot 2023-03-31 at 9 16 52 AM" src="https://user-images.githubusercontent.com/35968892/229174917-4072745c-1344-48dd-84d9-d74305701a18.png">

See how while using number dialpad the "client" toggle is visible.

Reply:
Ok got it.